src/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// PHP_CodeSniffer does not define an autoloader in its composer.json,
+// so we must invoke it ourselves.
+require __DIR__ . "/../vendor/squizlabs/php_codesniffer/autoload.php";
```

Comment:
Because there aren't any stable versions for 3.0 yet (2.x is the current stable version)